Tribler/Core/Modules/restapi/util.py

```diff
@@ -4,6 +4,16 @@
 from Tribler.Core.Modules.restapi import VOTE_SUBSCRIBE
 
 
+def convert_torrent_to_json(torrent):
```

Comment:
I would rather do that instead but unfortunately, we are not working with objects here. The database returns tuples and remote torrent results are returning dictionaries.